src/styles/main.scss

```
@@ -1,3 +1,61 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(300px, 1fr));
+  justify-content: center;
+  grid-auto-flow: row;
+}
+
+@media (max-width:899px) {
+  .container {
+    grid-template-columns: repeat(2, minmax(300px, 1fr));
+  }
+}
+
+@media (max-width:599px) {
+  .container {
+    grid-template-columns: minmax(300px, 1fr);
+  }
+}
+
+.container__block {
+  height: 300px;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  align-items: center;
```

```suggestion
  justify-content: center;
  align-content: center;
```
достаточно что б текст был по центру.